packages/core-ethereum/src/indexer/index.ts

```diff
@@ -416,17 +416,18 @@ class Indexer extends EventEmitter {
     this.indexEvent('channel-updated', [event.transactionHash])
     log('channel-updated for hash %s', event.transactionHash)
     const channel = await ChannelEntry.fromSCEvent(event, (a: Address) => this.getPublicKeyOf(a))
-
+    log(`Smart contract event`)
     log(channel.toString())
-    await this.db.updateChannel(channel.getId(), channel)
 
-    let prevState
+    let prevState: ChannelEntry
     try {
       prevState = await this.db.getChannel(channel.getId())
     } catch (e) {
       // Channel is new
```

Comment:
This is not great to swallow errors silently

Reply:
it's not an error, `db.getChannel` returns an error when a channel is not found, in this instance, we don't care, we are expecting that sometimes there will be no channel